openmrs/src/main/java/org/motechproject/openmrs/helper/EventHelper.java

```diff
@@ -195,4 +196,14 @@ private EventHelper() {
 
         return parameters;
     }
+
+    public static Map<String, Object> cohortMemberParameters(String cohortQueryUuid, CohortQueryReportMember member) {
```

Comment:
I think that we should send this event in task action logic

Additionally used string constants should be added to task package into appropriate class